environments/prod/prod.tfvars

```diff
@@ -513,13 +513,13 @@ frontends = [
     ]
   },
   {
-    name             = "jd-bureau"
-    custom_domain    = "juror-bureau.justice.gov.uk"
-    dns_zone_name    = "juror-bureau.justice.gov.uk"
-    backend_domain   = ["firewall-prod-int-palo-sdsprod.uksouth.cloudapp.azure.com"]
-    ssl_mode         = "AzureKeyVault"
-    certificate_name = "juror-bureau-justice-gov-uk"
-
+    name                         = "jd-bureau"
+    custom_domain                = "juror-bureau.justice.gov.uk"
+    dns_zone_name                = "juror-bureau.justice.gov.uk"
+    backend_domain               = ["firewall-prod-int-palo-sdsprod.uksouth.cloudapp.azure.com"]
+    ssl_mode                     = "AzureKeyVault"
+    certificate_name             = "juror-bureau-justice-gov-uk"
+    shutter_app                  = false
```

Comment:
what do you mean it has to be created as part of shuttering process? how are these supposed to work?

Reply:
we already have a **"@"** record pointing to frontdoor for apex domain(s). Shuttering process needs to verify the domain by adding same record as well.